class.jetpack.php

```diff
@@ -6832,6 +6844,55 @@ public static function build_raw_urls( $url ) {
 		return $url;
 	}
 
+	/**
+	 * Builds an URL using the jetpack.com/redirect service
+	 *
+	 * Note to WP.com: Changes to this method must be synced to wpcom
+	 *
+	 * @since 8.4.0
+	 *
+	 * @param string        $source The URL handler registered in the server
+	 * @param array|string  $args {
+	 *
+	 * 		Additional arguments to build the url
+	 *
+	 * 		@param string $site URL of the current site
```

Comment:
Just occurred to me that this could be optional and default to the current site URL. It would make it simpler to call it since most of the calls to this method only includes the URL of the current site...

Reply:
Addressed in https://github.com/Automattic/jetpack/pull/15140/commits/4beabdc1e9b9a7baa2bdea67a879f50a80102eaa